test/integration/mongodb-handshake/mongodb-handshake.test.ts

```diff
@@ -62,3 +64,89 @@ describe('MongoDB Handshake', () => {
     });
   });
 });
+context('when running against a server version >= 4.4', function () {
+  const HEARTBEAT_FREQUENCY_MS = 1000;
+  const MIN_HEARTBEAT_FREQUENCY_MS = 500;
+  const metadata = { requires: { mongodb: '>= 4.4', topology: '!load-balanced' } };
+  let cachedEnv: NodeJS.ProcessEnv;
+  let client: MongoClient;
+
+  beforeEach(async function () {
+    cachedEnv = { ...process.env };
+  });
+
+  afterEach(async function () {
+    await client.close().catch(() => null);
+    process.env = cachedEnv;
+  });
+
+  context('in a FaaS environment', function () {
+    beforeEach(function () {
+      process.env.AWS_EXECUTION_ENV = 'AWS_Lambda_test';
+      process.env.AWS_REGION = 'test_region';
+      client = this.configuration.newClient(
+        {},
+        {
+          heartbeatFrequencyMS: HEARTBEAT_FREQUENCY_MS,
+          minHeartbeatFrequencyMS: MIN_HEARTBEAT_FREQUENCY_MS
+        }
+      );
+    });
+
+    it('uses the polling protocol', metadata, async function () {
+      await client.connect();
+      const servers = client?.topology.s?.servers;
+      expect(servers).to.exist;
+      for (const server of servers.values()) {
+        const symbolMap = {};
```

Comment:
Here and also in the other test instead of constructing an object and putting all the symbols in it, we could just use `find` on the iterable:

```ts
const monitorSymbol = Object.getOwnPropertySymbols(server).find(s => s.toString() === 'Symbol(monitor)');
```

Reply:
We have the [getSymbolFrom](https://github.com/mongodb/node-mongodb-native/blob/main/test/tools/utils.ts#L115) helper that I think helps here